dll/win32/msgina/lang/ja-JP.rc

```diff
@@ -177,39 +177,39 @@ BEGIN
     DEFPUSHBUTTON "OK", IDOK, 115, 179, 50, 14
 END
 
-IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 208, 122
+IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 208, 121
 STYLE NOT WS_VISIBLE | DS_CENTER | DS_SHELLFONT | WS_BORDER | WS_POPUP
 FONT 9, "MS UI Gothic"
 BEGIN
-    CONTROL IDI_DLG_BG, IDC_DLG_BG, "STATIC", SS_BITMAP | SS_REALSIZECONTROL, 0, 0, 208, 122
-    CONTROL IDI_ROSLOGO_FLAG, IDC_ROSLOGO_WHITE, "STATIC", SS_BITMAP, 180, 2, 32, 26
-    PUSHBUTTON "休止(&H)", IDC_BUTTON_HIBERNATE, 30093, 30049, 19, 19, BS_OWNERDRAW | WS_DISABLED
+    CONTROL "", IDC_SHUTDOWN_DLG_BG, "STATIC", SS_OWNERDRAW, 0, 0, 208, 121
+    CONTROL "", IDC_SHUTDOWN_DLG_FLAG, "STATIC", SS_OWNERDRAW, 180, 1, 32, 26
+    PUSHBUTTON "休止(&H)", IDC_BUTTON_HIBERNATE, 30093, 30049, 19, 19, BS_OWNERDRAW
     PUSHBUTTON "スリープ(&P)", IDC_BUTTON_SLEEP, 38, 46, 19, 19, BS_OWNERDRAW | WS_GROUP
     PUSHBUTTON "シャットダウン(&S)", IDC_BUTTON_SHUTDOWN, 95, 46, 19, 19, BS_OWNERDRAW
     PUSHBUTTON "再起動(&R)", IDC_BUTTON_REBOOT, 152, 46, 19, 19, BS_OWNERDRAW
     PUSHBUTTON "キャンセル", IDCANCEL, 162, 103, 40, 12, BS_FLAT | WS_GROUP
-    LTEXT "ReactOS のシャットダウン", IDC_TURN_OFF_STATIC, 4, 9, 130, 20
-    CTEXT "スリープ", IDC_SLEEP_STATIC, 22, 70, 51, 11
-    CTEXT "シャットダウン", IDC_SHUTDOWN_STATIC, 79, 70, 51, 11
-    CTEXT "再起動", IDC_RESTART_STATIC, 136, 70, 51, 11
+    CONTROL "", IDC_TURN_OFF_STATIC, "STATIC", SS_OWNERDRAW, 4, 8, 130, 20
+    CONTROL "", IDC_SLEEP_STATIC, "STATIC", SS_OWNERDRAW, 22, 70, 51, 11
+    CONTROL "", IDC_SHUTDOWN_STATIC, "STATIC", SS_OWNERDRAW, 79, 70, 51, 11
+    CONTROL "", IDC_RESTART_STATIC, "STATIC", SS_OWNERDRAW, 136, 70, 51, 11
 END
 
-IDD_SHUTDOWN_FANCY_LONG DIALOGEX 0, 0, 264, 122
+IDD_SHUTDOWN_FANCY_LONG DIALOGEX 0, 0, 264, 121
```

Comment:
```suggestion
IDD_SHUTDOWN_FANCY_LONG DIALOGEX 0, 0, 264, 122
```
OK I don't want to say, but the sizes of the pushbuttons and some other controls appear to be "off" compared to all the other languages.